primitives/runtime/src/lib.rs

```diff
@@ -285,6 +285,28 @@ pub fn storage_value_key(pallet_prefix: &str, value_name: &str) -> StorageKey {
 	StorageKey(final_key)
 }
 
+/// Can be use to access the runtime storage key of a `StorageDoubleMap`.
+pub trait StorageMapKeyProvider {
+	// The name of the variable that holds the `StorageDoubleMap`
```

Comment:
Let's change it to `///` maybe (and below as well)

Reply:
Done.